Обновление README.md

manmolecular opened the pull request:
Немного обновил _readme_ файл, учитывая текущий процесс разработки:

1. Добавил скриншоты
2. Добавил описание
3. Добавил общую информацию

manmolecular committed:
[+] Update README.md

Немного обновил readme файл, учитывая текущий процесс разработки.

imanashev commented on README.md:
```diff
@@ -1,12 +1,33 @@
-![alt tag](http://adanai.com/wp-content/uploads/2012/12/Pong.png)
+**Скриншоты игрового процесса:**
+![alt tag](https://pp.userapi.com/c841326/v841326566/31fcb/muViS6lubTU.jpg)
+![alt tag](https://pp.userapi.com/c840738/v840738343/19650/W3pjzG-3CjQ.jpg)
```
Давай без скрина c уровнем `ILDAR` :smiley:

vosafonov replied:
:D

manmolecular replied:
Да в общем не страшно было бы :wink:
Но хорошо.

imanashev commented on README.md:
```diff
@@ -1,12 +1,33 @@
+
+**Что мы используем при разработке:**
+- Cocos2d-x
+- CLion
```
Не знаю, стоит ли писать тут про CLion? А вот CMake, google-tests думаю нужно

manmolecular replied:
Я думаю, ничего страшного, если мы укажем и саму среду разработки - по крайней мере, мне зачастую интересен весь инструментарий, который используется при разработке чего-то, что мне интересно. Насчет CMake, google etc. согласен, да - хорошо.

imanashev commented on README.md:
```diff
@@ -1,12 +1,33 @@
+
+**Что у нас есть:**
+- Полноценный играбельный арканоид
+- Визуальные эффекты
```
Смотря что понимать под визуальными эффектами. В моем понимании это эффекты столкновений, взрывы.

manmolecular replied:
Хм, я предполагал что-то в общем попроще говоря "визуальные эффекты", но, наверное, да, тоже соглашусь. Я просто прирожденный маркетолог, не обращайте внимания :relieved:

imanashev commented on README.md:
```diff
@@ -1,12 +1,33 @@
+- Полноценный играбельный арканоид
+- Визуальные эффекты
+- Звуковые эффекты
+- Генерация уровней
```
Еще нет же генерации

vosafonov replied:
Документация работает наперёд :)

manmolecular replied:
Оп, исправлю.

vosafonov commented:
@tsu-trpo/ping-pong-team, круто! Мне нравится ваш подход ;)

manmolecular committed:
[+] Исправления по комментариям

- Удалил один скриншот
- Добавил инструментарий
- Убрал лишнее в готовом
- Перенёс всё куда нужно (вроде бы)

manmolecular commented:
Учел пожелания и внёс исправления.

UPD: Забыл описать сборку проекта. Добавлю в ближайшее время.

mshsmlv commented:
а я думала мы ping-pong пишем:D, а не арканоид

manmolecular commented:
@MashaSamoylova, ну, на данный момент это скорее арканоид, и, я думаю, таким он и будет. Но никогда не поздно обновить/переписать, так что в случае чего - перепишем/изменим :smiley_cat:

manmolecular committed:
[+] Добавил описание сборки проекта

- Описал сборку проекта под linux
- Добавил bash-команды
- Не обошлось без маркетинга

manmolecular committed:
[+] Небольшая ремарка

- Комментарий по поводу первой команды

manmolecular commented:
Описал сборку проекта. В принципе, там всё элементарно, конечно, но даже там можно собрать какой-нибудь шелл-скрипт в стиле run.sh, чтобы вообще ничего ручками не вбивать - хотя, смысла нет, наверное, ради четырех-то команд?

manmolecular committed:
[+] Небольшое дополнение к bash-сборке

- Собирал с нуля и понял, что пропустил пару команд

imanashev reviewed (approved):
+1

vosafonov commented on README.md:
```diff
@@ -1,12 +1,53 @@
+git submodule update --recursive --init
+cd SUPER-PING-PONG/
+cmake .
+make
```
Тут точно все ок? 1) нам вроде нужен `cocos` и компиляция/запуск с помощью команды `cocos run -p linux`, 2)  если даже можно обойтись без команды из п.1., то `cmake` обычно выполняется в папке `build`, чтобы не создавать лишние файлы в ппроекте. Т.е. должно быть как-то так:
```
cd SUPER-PING-PONG
mkdir build && cd build
cmake ..
make
```

mshsmlv replied:
тут еще такая проблема, что вручную через 
```
mkdir build && cd build
cmake ..
make
```
почему-то не собирается. Помнишь, Вадим, тогда, когда Cmake писали, не могли собрать.
а вот через 
```
cocos run -p linux 
```
всё ок

mshsmlv replied:
а, нет, сейчас всё отлично собралось.

manmolecular replied:
Так, разобрался, да - учел п.1 и переписал запуск попроще. С п.2 разберусь, если всё будет ок - добавлю вариативный запуск в readme.

manmolecular replied:
Разобрался с п.2 - всё собралось, добавил этот вариант сборки как альтернативный в случае, если что-то будет не так с запуском через команду cocos.

vosafonov replied:
Отлично!

manmolecular committed:
[+] Исправил запуск с учетом замечаний

- Учел п. 1 и упростил запуск

manmolecular committed:
[+] submodule

- Забыл про подключаемые библиотеки

manmolecular committed:
[+] Альтернативный вариант запуска

- Добавил альтернативный вариант запуска без cocos
- Немного изменил структуру документа

vosafonov reviewed (approved):
+1
